### PR TITLE
Addressing opsdir comments

### PR DESCRIPTION
**Pull request by seanturner**
Addressing [opsdir comments](https://mailarchive.ietf.org/arch/msg/rtcweb/6a-xbYS2dihJXRRqaGoSNDQVRNQ/).

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -205,11 +205,11 @@
           When <xref target="RFC8829"/> was published, inconsistencies regarding BUNDLE
           <xref target="RFC8843"/> operation were identified with regard to
           both the specification text as well as implementation behavior. The
-          former concern was addressed via an update to <xref target="RFC8843"/>.
+          former concern was addressed via an update to BUNDLE, see <xref target="RFC9143"/>.
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml:208]:
```suggestion
          former concern was addressed via an update to BUNDLE (see <xref target="RFC9143"/>).
```

**Review by ekr — APPROVED** [review]
LGTM modulo the change below. @juberti @fluffy

**Review by juberti — APPROVED** [review]
This seems like a good clarification, but there is still some confusion with 8843 vs 9143 references. I've added some suggested text to resolve.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -205,11 +205,11 @@
           When <xref target="RFC8829"/> was published, inconsistencies regarding BUNDLE
           <xref target="RFC8843"/> operation were identified with regard to
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:206]: I'm guessing we want to leave this one as RFC8843?

Comment (seanturner) [reply draft-ietf-rtcweb-jsep.xml:206]: I think this one we do want to leave alone because it is referring to the "old" version.

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:206]: OK. I will update accordingly and merge.

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:206]: PTAL at the updated text for this PR

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:206]: @seanturner

**Pull request merged**